File: backend/src/main/java/xyz/engsmyre/moviedescriptiongame/tmdb/repository/MovieRepository.java
package xyz.engsmyre.moviedescriptiongame.tmdb.repository;

import java.util.List;
import xyz.engsmyre.moviedescriptiongame.tmdb.domain.Movie;

public interface MovieRepository {
    List<Movie> getPopularMoviesFromPage(int id);  // TODO Not hardcoded popular
    int getPopularMoviesPageCount();
}

File: backend/src/main/java/xyz/engsmyre/moviedescriptiongame/tmdb/repository/MovieRepository.kt
package xyz.engsmyre.moviedescriptiongame.tmdb.repository

import xyz.engsmyre.moviedescriptiongame.tmdb.domain.Movie

interface MovieRepository {
    fun getPopularMoviesFromPage(page : Int): List<Movie?>? // TODO Not hardcoded popular
    val popularMoviesPageCount: Int
}